package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "prepend-file",
-  "version": "1.3.1",
+  "name": "prepend-to-js-file",
```

Comment:
Don't do unrelated changes.

Reply:
From the PR: 
> Hence you might want to cherry-pick instead of executing a straight merge of this work.
😉 

Didn't have enough time to do a cleaned-up PR, alas.